cli/lsp/language_server.rs

```diff
@@ -902,6 +930,11 @@ impl Inner {
     self.performance.measure(mark);
   }
 
+  async fn refresh_npm_specifiers(&mut self) {
+    let package_reqs = self.documents.npm_package_reqs();
+    let _ = self.npm_resolver.set_package_reqs(package_reqs).await;
```

Comment:
Shouldn't there be a log here if this function is fallible?

Reply:
Actually, there's no harm if I just log this. I will do that.